tracpro/client.py

```diff
@@ -51,6 +51,20 @@ def get_contacts_in_groups(self, group_uuids, deleted=None):
 
         return contacts
 
+    def check_urn_exists(self, urn, uuid):
+        """
+        Check to see if a urn is already taken by
+        a contact in RapidPro. exclude contacts with matching uuids,
+        indicating that the contact is being updated
+        """
+        contacts = self.get_contacts(urn=urn)
+        if len(list(contacts)) > 0 and contacts[0].uuid != uuid:
+            return True
+        elif len(list(contacts)) > 0 and contacts[0].uuid == uuid:
+            return False
+        else:
+            return False
+
```

Comment:
+1 on shortening to return value of the expression.  

Otherwise, I agree that we should be able to just write `if contacts`, and started off doing so, but I get a key error from line 81 of the same page.

Reply:
That sounds like a bug in our client code.

Maybe apply this change and try again?

        diff --git a/tracpro/client.py b/tracpro/client.py
        index c1b5f15..8655147 100644
        --- a/tracpro/client.py
        +++ b/tracpro/client.py
        @@ -70,3 +70,9 @@ class TracProCursorQuery(CursorQuery):

             def __getitem__(self, index):
                 return self._get_result()[index]
        +
        +    def __len__(self):
        +        return len(self._get_result())
        +
        +    def __nonzero__(self):
        +        return len(self._get_result()) != 0